src/Microsoft.Identity.Client/ApiConfig/AcquireTokenForClientParameterBuilder.cs

```diff
@@ -58,12 +58,11 @@ public AcquireTokenForClientParameterBuilder(IConfidentialClientApplication conf
         }
 
         /// <summary>
-        /// Specifies if the confidential client application should force refreshing the
-        /// token from the application token cache. By default the token is taken from the
-        /// the application token cache (forceRefresh=false)
+        /// Specifies if the token request will ignore the access token in the application token cache
```

Comment:
I liked @bgavrilMS 's suggestion here and took it from the original PR, moving the <param> comment to the summary and shortening the param comment.